Mixpanel/Mixpanel.h

```diff
@@ -201,6 +201,19 @@
 
 /*!
  @property
+ 
+ @abstract
+ If set, determines the background color of mini notifications.
+
+ @discussion
+ If this isn't set, we default to either the color of the UINavigationBar of the top 
+ UINavigationController that is showing when the notification is presented, the 
+ UINavigationBar default color for the app or the UITabBar default color.
+ */
+@property (atomic) UIColor* miniNotificationBackgroundColor;
+
```

Comment:
Sorry, I thought `strong` was the default?

From the [docs](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html#//apple_ref/doc/uid/TP40011210-CH5-SW30):

>There’s no need to specify the strong attribute explicitly, because it is the default.